starky/src/lookup.rs

```diff
@@ -402,12 +398,24 @@ impl<F: Field> Column<F> {
 
 pub(crate) type ColumnFilter<'a, F> = (&'a [Column<F>], &'a Option<Filter<F>>);
 
+/// A `Lookup` defines a set of `columns`` whose values should appear in a
```

Comment:
```suggestion
/// A `Lookup` defines a set of `columns` whose values should appear in a
```

Also, should these be [Lookup] etc, as in the other comments?